src/executive/executive.cpp

```diff
@@ -255,6 +255,14 @@ BehavioralExecutive::runSweep()
     /* Transition to Idle */
     if (eventDict[STOP]) currentState = IDLE_STATE;
 
+	/* Perform the concluding switch to idle */
+	if (eventDict[NO_HUMAN] and eventDict[CONCLUDE_SWEEP]) {
```

Comment:
should this transition be before line 245 otherwise it might go to explore again?

Before this part
```
/* Transition to Explore */
if (eventDict[NO_HUMAN] and not eventDict[USER_CONTROL])
{
    ros::Duration(2.0).sleep();
    currentState = EXPLORE_STATE;
}
```

Reply:
Yeah I'm actually not sure why this didn't occur during the testing.

Reply:
Hmm this actually doesn't work for some reason
![image](https://user-images.githubusercontent.com/29968424/139873348-43e11a52-2558-413a-880c-c083c2d65ba9.png)

Reply:
Oh what I had before is actually correct because I'm only setting the current state variable, not exiting the function. HOwever just to be super safe I will add this: 
```
if (eventDict[NO_HUMAN] and not eventDict[USER_CONTROL] and not eventDict[CONCLUDE_SWEEP])
    {
        ROS_INFO_STREAM((eventDict[NO_HUMAN] and not eventDict[USER_CONTROL]) << " THis is going to explore? ");
        ros::Duration(2.0).sleep();
        currentState = EXPLORE_STATE;
    }
```

Reply:
cool